packages/core/src/credential/Credential.spec.ts

```diff
@@ -532,11 +527,10 @@ describe('Credential', () => {
     })
   })
   it('verify credentials signed by a light DID', async () => {
-    const { keypair, sign } = makeSigningKeyTool('ed25519')
-    identityDave = await LightDidDetails.fromIdentifier(
-      encodeAddress(keypair.publicKey, ss58Format),
-      'ed25519'
-    )
+    const { sign, authentication } = makeSigningKeyTool('ed25519')
+    identityDave = await Did.createDetails({
```

Comment:
If this function creates lightdid details, I think the namespace or function name should reflect that.
e.g. `Did.createLight`

Reply:
The more I think about it, the more I would prefer namespacing. `LightDid.createDetails` in this case. Or with the full did: `FullDid.Chain.getStoreTx`. What do you think?

Reply:
I think the `LightDid` part rather belongs in the name of the function itself, I don’t see many other candidates for including in the `LightDid` namespace.

Reply:
Done in 0d367a30